UI/src/view/linear_asset/piecewiseLinearAssetFormElements.js

```diff
@@ -45,9 +45,9 @@
       }
     }
 
-    function singleValueEditElement(currentValue, sideCode, input) {
-      var withoutValue = _.isUndefined(currentValue) ? 'checked' : '';
-      var withValue = _.isUndefined(currentValue) ? '' : 'checked';
+    function singleValueEditElement(currentValue, sideCode, input, id) {
+      var withoutValue = _.isNull(id) ? 'checked' : '';
```

Comment:
This works on creation? Because when the asset is created we don't have id

Reply:
this is called for form creation after selecting an asset, not on value input on the form. the thing is that there are assets without values, so the check needs to be on id